evalml/automl/automl_search.py

```diff
@@ -402,24 +401,14 @@ def search(self, X, y, data_checks="auto", show_iteration_plot=True):
             except NameError:
                 show_iteration_plot = False
 
-        # make everything ww objects
-        if not isinstance(X, ww.DataTable):
-            logger.warning("`X` passed was not a DataTable. EvalML will try to convert the input as a Woodwork DataTable and types will be inferred. To control this behavior, please pass in a Woodwork DataTable instead.")
-            X = _convert_to_woodwork_structure(X)
-
-        text_column_vals = X.select('natural_language')
+        text_column_vals = self.X_train.select('natural_language')
         text_columns = list(text_column_vals.to_dataframe().columns)
         if len(text_columns) == 0:
             text_columns = None
 
-        if not isinstance(y, ww.DataColumn):
-            logger.warning("`y` passed was not a DataColumn. EvalML will try to convert the input as a Woodwork DataTable and types will be inferred. To control this behavior, please pass in a Woodwork DataTable instead.")
-            y = _convert_to_woodwork_structure(y)
```

Comment:
I removed these warnings because we were planning on doing so (#1555) and because it was easier than fixing the failing test coverage for them 😅